packages/combobox/src/ComboboxOption/ComboboxOption.styles.ts

```diff
@@ -61,3 +58,63 @@ export const disallowPointer = css`
 export const displayNameStyle = (isSelected: boolean) => css`
   font-weight: ${isSelected ? 'bold' : 'normal'};
 `;
+
+export const iconThemeStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.gray.dark1};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.gray.base};
+  `,
+};
+
+export const iconHighlightedStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.blue.dark1};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.blue.light3};
+  `,
+};
+
+export const iconDisabledStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.gray.light1};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.gray.dark1};
+  `,
+};
+
+export const checkMarkThemeStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.blue.base};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.blue.light1};
+  `,
+};
+
+export const checkMarkDisabledStyles: Record<Theme, string> = {
+  [Theme.Light]: css`
+    color: ${palette.gray.light1};
+  `,
+  [Theme.Dark]: css`
+    color: ${palette.gray.dark1};
+  `,
+};
+
+export const multiselectIconPosition = css`
+  > div {
+    > div:first-of-type {
+      align-self: baseline;
+    }
+  }
+`;
+export const multiselectIconLargePosition = css`
+  > div {
+    > div:first-of-type {
+      height: 28px;
+    }
+  }
+`;
```

Comment:
This feels brittle, but I'm not sure what the "right" way to do this would be

Reply:
don't we have a util for this?

Reply:
Can you elaborate @bruugey?

Reply:
createUniqueClassName?